src/Plugins/ITKImageProcessing/src/ITKImageProcessing/Filters/Algorithms/ITKImportFijiMontage.cpp

```diff
@@ -269,7 +269,7 @@ class IOHandler
         // rename grayscale array to reflect original
         {
           auto& gray = m_DataStructure.getDataRefAs<IDataArray>(imageDataPath.replaceName("gray" + imageDataPath.getTargetName()));
-          if(!gray.canRename(imageDataPath.getTargetName()))
+          if(gray.canRename(imageDataPath.getTargetName()) != 1)
```

Comment:
```suggestion
          if(gray.canRename(imageDataPath.getTargetName()) == false)
```
Keep the boolean typing.

Reply:
I see the change in DataObject::canRename documentation that this is referencing, but the implementation in DataObject.cpp is unchanged. There are no special return codes to reference here.